pythonforandroid/archs.py

```diff
@@ -93,20 +123,20 @@ def get_env(self, with_flags_in_cc=True):
                 'installed. Exiting.')
```

Comment:
The `{}` string argument is not rendered.
This is minor since it's out of context, but FWIW: https://github.com/kivy/python-for-android/pull/1561